src/main/java/net/sf/jabref/logic/formatter/bibtexfields/PageNumbersFormatter.java

```diff
@@ -18,6 +18,9 @@
 
     private static final Pattern PAGES_PATTERN = Pattern.compile("\\A(\\d+)-{1,2}(\\d+)\\Z");
 
+    private static final String REJECT_LITERALS = "[^0-9,\\-\\+]";
+    private static final String REPLACE = "$1--$2";
```

Comment:
Replace is not very descriptive. What about PagePattern or if you want ReplacePattern?